src/sentry/static/sentry/app/components/deviceName.jsx

```diff
@@ -0,0 +1,30 @@
+import PropTypes from 'prop-types';
+import React from 'react';
+
+export default class DeviceName extends React.Component {
+  static propTypes = {
+    children: PropTypes.string,
+  };
+
+  constructor(...args) {
+    super(...args);
+    this.state = {
+      deviceNameMapper: null,
+    };
+  }
+
+  componentWillMount() {
```

Comment:
Maybe worth adding a comment that `ios-device-list` is a huge lib and should be loaded this way?